### PR TITLE
Update of DNS list

### PR DESCRIPTION
**Pull request by TheHasagi**
Update of DNS list regarding #369

### CODE REVIEW

File: 02.general/17.dns-providers/docs.en.md [hunk 1]
```diff
@@ -243,8 +243,8 @@ OpenDNS servers that provide adult content blocking
 |----------------|----------------------------------------------------|----------------|
 | DNS, IPv4      | `1.1.1.1` and `1.0.0.1`           | <a href="sdns://AAAAAAAAAAAABzEuMS4xLjE">Add to AdGuard</a> |
 | DNS, IPv6      | `2606:4700:4700::1111` and `2606:4700:4700::1001`            | <a href="sdns://AAAAAAAAAAAAFlsyNjA2OjQ3MDA6NDcwMDo6MTExMV0">Add to AdGuard</a> |
-| DNS-over-HTTPS, IPv4 | `https://dns.cloudflare.com/dns-query` | <a href="sdns://AgcAAAAAAAAABzEuMC4wLjGgENk8mGSlIfMGXMOlIlCcKvq7AVgcrZxtjon911-ep0cg63Ul-I8NlFj4GplQGb_TTLiczclX57DvMV8Q-JdjgRgSZG5zLmNsb3VkZmxhcmUuY29tCi9kbnMtcXVlcnk">Add to AdGuard</a> |
-| DNS-over-HTTPS, IPv6 | `https://dns.cloudflare.com/dns-query` | <a href="sdns://AgcAAAAAAAAAGVsyNjA2OjQ3MDA6NDcwMDo6MTExMV06NTOgENk8mGSlIfMGXMOlIlCcKvq7AVgcrZxtjon911-ep0cg63Ul-I8NlFj4GplQGb_TTLiczclX57DvMV8Q-JdjgRgSZG5zLmNsb3VkZmxhcmUuY29tCi9kbnMtcXVlcnk">Add to AdGuard</a> |
+| DNS-over-HTTPS, IPv4 | `https://cloudflare-dns.com/dns-query` | <a href="sdns://AgcAAAAAAAAABzEuMC4wLjGgENk8mGSlIfMGXMOlIlCcKvq7AVgcrZxtjon911-ep0cg63Ul-I8NlFj4GplQGb_TTLiczclX57DvMV8Q-JdjgRgSZG5zLmNsb3VkZmxhcmUuY29tCi9kbnMtcXVlcnk">Add to AdGuard</a> |
```
Comment (admitrevskiy) [inline 02.general/17.dns-providers/docs.en.md]: SDNS stamps do not match hosts

Comment (admitrevskiy) [reply 02.general/17.dns-providers/docs.en.md]: BTW we need to fix ru version also

File: 02.general/17.dns-providers/docs.en.md [hunk 2]
```diff
@@ -260,7 +260,7 @@ Regular and DNS-over-TLS servers which provide protection from phishing and spyw
 | Protocol       | Address                                            |                |
 |----------------|----------------------------------------------------|----------------|
 | DNS, IPv4      | `9.9.9.9` and `149.112.112.112`                                          | <a href="sdns://AAAAAAAAAAAABzkuOS45Ljk">Add to AdGuard</a> |
-| DNS-over-TLS   | `tls://dns.quad9.net`                                           | <a href="sdns://AwAAAAAAAAAAAAANdGxzOi8vOS45LjkuOQ">Add to AdGuard</a> |
+| DNS-over-TLS   | `tls://dns.quad9.net/dns-query`                                           | <a href="sdns://AwAAAAAAAAAAAAANdGxzOi8vOS45LjkuOQ">Add to AdGuard</a> |
```
Comment (admitrevskiy) [inline 02.general/17.dns-providers/docs.en.md]: This one looks wrong

File: 02.general/17.dns-providers/docs.en.md (hunk 1 above)
Comment (TheHasagi) [reply 02.general/17.dns-providers/docs.en.md]: Yep, asked your review to investigate and help with correct stamps.

Comment (admitrevskiy) [reply 02.general/17.dns-providers/docs.en.md]: Stamps are still not valid

File: 02.general/17.dns-providers/docs.ru.md [hunk 3]
```diff
@@ -230,7 +230,7 @@ OpenDNS серверы с функцией блокировки контента
 |----------------|----------------------------------------------------|----------------|
 | DNS, IPv4      | `8.8.8.8` и `8.8.4.4`           | <a href="sdns://AAAAAAAAAAAABzguOC44Ljg">Добавить в AdGuard</a> |
 | DNS, IPv6      | `2001:4860:4860::8888` и `2001:4860:4860::8844`            | <a href="sdns://AAAAAAAAAAAAFlsyMDAxOjQ4NjA6NDg2MDo6ODg4OF0">Добавить в AdGuard</a> |
-| DNS-over-HTTPS | `https://dns.google/dns-query` | <a href="sdns://AgUAAAAAAAAAACAe9iTP_15r07rd8_3b_epWVGfjdymdx-5mdRZvMAzBuQ5kbnMuZ29vZ2xlLmNvbQ0vZXhwZXJpbWVudGFs">Добавить в AdGuard</a> |
+| DNS-over-HTTPS | `https://dns.google/dns-query` | <a href="sdns://AgcAAAAAAAAAAAAKZG5zLmdvb2dsZQovZG5zLXF1ZXJ5">Добавить в AdGuard</a> |
```
Comment (admitrevskiy) [inline 02.general/17.dns-providers/docs.ru.md]: Не хватает изменения для cloudflare в ру версии

File: 02.general/17.dns-providers/docs.en.md [hunk 4]
```diff
@@ -260,7 +260,7 @@ Regular and DNS-over-TLS servers which provide protection from phishing and spyw
 | Protocol       | Address                                            |                |
 |----------------|----------------------------------------------------|----------------|
 | DNS, IPv4      | `9.9.9.9` and `149.112.112.112`                                          | <a href="sdns://AAAAAAAAAAAABzkuOS45Ljk">Add to AdGuard</a> |
-| DNS-over-TLS   | `tls://dns.quad9.net`                                           | <a href="sdns://AwAAAAAAAAAAAAANdGxzOi8vOS45LjkuOQ">Add to AdGuard</a> |
+| DNS-over-TLS   | `tls://dns.quad9.net/dns-query`                                           | <a href="sdns://AwAAAAAAAAAAAAAddGxzOi8vZG5zLnF1YWQ5Lm5ldC9kbnMtcXVlcnk">Add to AdGuard</a> |
```
Comment (admitrevskiy) [inline 02.general/17.dns-providers/docs.en.md]: Still looks weird :)

**Pull request closed without merging**